PyInstaller/utils/win32/winutils.py

```diff
@@ -12,8 +12,12 @@
 Utilities for Windows platform.
 """
 
+import time
```

Comment:
Keep a gap between stdlib/3rd party/local import groups
```suggestion
import time

```